conf/functions_worker.yml

```diff
@@ -93,6 +93,8 @@ schedulerClassName: "org.apache.pulsar.functions.worker.scheduler.RoundRobinSche
 functionAssignmentTopicName: "assignments"
 failureCheckFreqMs: 30000
 rescheduleTimeoutMs: 60000
+# frequency at which to check if cluster needs rebalancing (15min default interval)
+rebalanceCheckFreqSec: 900
```

Comment:
Do we want to turn it on by default? Also is 15 min too frequent

Reply:
Set it 1 hour.  Is there any harm to enable by default?